x/auth/spec/05_vesting.md

```diff
@@ -22,20 +22,18 @@
 
 ## Intro and Requirements
 
-This specification describes the vesting account implementation for the Cosmos Hub.
-The requirements for this vesting account is that it should be initialized
-during genesis with a starting balance `X` and a vesting end time `T`.
+This specification defines the vesting account implementation that is used by the Cosmos Hub. The requirements for this vesting account is that it should be initialized during genesis with a starting balance `X` and a vesting end time `T`. A vesting account may be initialized with a vesting start time `T'` and a number of vesting periods `P`. If a vesting start time is included, the vesting period will not begin until start time is reached. If vesting periods are included, the vesting will occur over the specified number of periods.
```

Comment:
```suggestion
This specification defines the vesting account implementation that is used by the Cosmos Hub. The requirements for vesting accounts is that they should be initialized during genesis with a starting balance `X` and a vesting end time `T`. A vesting account may be initialized with a vesting start time `T'` or a number of vesting periods `P`. If a vesting start time is included, the vesting period will not begin until start time is reached. If vesting periods are included, the vesting will occur over the specified number of periods.
```

Reply:
Also, I realized we use `T` for multiple values, let's change them to `ST` for start time and `ET` for end time?